### PR TITLE
Revise daily_cancities script to use processed ERA5 data

### LINKED ISSUE #20: ERA5 evspsblpot data has mixed and mostly negative values
Discovered this while working on a Dryness Index metric in xclim:
```
ds.evspsblpot.values
Out [n]:
array([[ 2.5760319e-06, -3.3353365e-06, -3.1583652e-06, ...,
        -1.5285735e-06, -7.4252091e-07, -1.8427075e-06],
       [-4.3911914e-06, -5.0908639e-06, -6.6358170e-06, ...,
        -2.1413846e-06, -3.1031943e-06, -4.4150788e-06],
       [-2.6260412e-07, -1.3417203e-07, -3.0068904e-07, ...,
        -8.3346769e-08, -4.0213813e-08, -3.6980322e-07],
       [-2.7174301e-06, -2.3758899e-07, -1.2333340e-06, ...,
        -3.5071828e-06, -2.7833125e-06, -2.6600276e-06],
       [-5.6763719e-07, -3.7722674e-07, -2.4505559e-07, ...,
        -1.0922088e-07, -1.4692718e-08,  3.4909103e-08]], dtype=float32)
```
These values should all be positive or 0.

### PR DESCRIPTION
**Pull request by Zeitsperre**
- [x] added a ``.md5`` checksum file generated with ``make_check_sums.py``

Fixes #20

### CODE REVIEW

File: ERA5/construct_era5_daily_cancities_revised.py [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+from xclim.core.units import convert_units_to
+
+logging.basicConfig(level=logging.INFO)
+
+# if len(sys.argv) == 2:
+#     base_path = Path(sys.argv[-1])
+# else:
+#     base_path = Path().cwd()
+
+base_path = Path("/tank/scenario")
```
Comment (aulemahal) [inline ERA5/construct_era5_daily_cancities_revised.py]:
```suggestion
if len(sys.argv) == 2:
    base_path = Path(sys.argv[-1])
else:
    base_path = Path().cwd()
```
Achtung!

File: ERA5/construct_era5_daily_cancities_revised.py [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+    tasmax.attrs.update(
+        standard_name="air_temperature",
+        long_name="Mean daily surface temperature",
+        units="K",
+        cell_methods="time: maximum within days",
+    )
+
+    if "pr" not in dly.data_vars:
+        # Total precip flux in kg m-2 s-1
+        pr = hrly.pr.resample(time="D").sum()
```
Comment (aulemahal) [inline ERA5/construct_era5_daily_cancities_revised.py]:
```suggestion
        pr = hrly.pr.resample(time="D").mean()
```
If it is a flux, then the resampling should be a mean, no?

File: ERA5/construct_era5_daily_cancities_revised.py [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+        standard_name="precipitation_flux",
+        long_name="Mean daily precipitation flux",
+        units="kg m-2 s-1",
+        cell_methods="time: mean within days",
+        description="Total precipitation thickness converted to mass flux using a water density of 1000 kg/m³.",
+    )
+
+    if "evspsblpot" not in dly.data_vars:
+        # Total Potential Evapotranspiration flux in kg m-2 s-1
+        evspsblpot = hrly.evspsblpot.resample(time="D").sum()
```
Comment (aulemahal) [inline ERA5/construct_era5_daily_cancities_revised.py]:
```suggestion
        evspsblpot = hrly.evspsblpot.resample(time="D").mean()
```
Same as above.

File: ERA5/construct_era5_daily_cancities_revised.py [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+        standard_name="water_potential_evaporation_flux",
+        long_name="Mean daily potential evaporation flux",
+        units="kg m-2 s-1",
+        cell_methods="time: mean within days",
+        description="Total potential evaporation thickness converted to mass flux using a water density of 1000 kg/m³.",
+    )
+
+    if "prsn" not in dly.data_vars:
+        # Total Solid Precipitation flux in kg m-2 s-1
+        prsn = hrly.prsn.resample(time="D").sum()
```
Comment (aulemahal) [inline ERA5/construct_era5_daily_cancities_revised.py]:
```suggestion
        prsn = hrly.prsn.resample(time="D").mean()
```
Same same.

File: ERA5/construct_era5_daily_cancities_revised.py [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+    if "vas" not in dly.data_vars:
+        vas = hrly.vas.resample(time="D").mean()
+    else:
+        vas = dly.vas
+
+    windmag, _ = xc.atmos.wind_speed_from_vector(uas=hrly.uas, vas=hrly.vas)
+    sfcWind = windmag.resample(time="D").mean()
+    wsgsmax = windmag.resample(time="D").max()
+
+    _, sfcWindfromdir = xc.atmos.wind_speed_from_vector(uas=uas, vas=vas)
```
Comment (aulemahal) [inline ERA5/construct_era5_daily_cancities_revised.py]: In theory, `uas` and `vas` can't be resampled directly.  See example in xscen : https://github.com/Ouranosinc/xscen/blob/331e2ecc78fdf9cb96ac903319f7371ea7067f47/xscen/extract.py#L470

In summary, the real daily components are reconstructed from the resampled magnitude and the angle of the resampled components.

Comment (aulemahal) [reply ERA5/construct_era5_daily_cancities_revised.py]: It is not primordial to do it correctly here, but it feels like a low-hanging fruit.

**Review by aulemahal — APPROVED** [review]
Thanks!
I believe the flux resampling logic is not exact and that the wind resampling could be improved.

File: ERA5/construct_era5_daily_cancities_revised.py (hunk 1 above)
Comment (Zeitsperre) [reply ERA5/construct_era5_daily_cancities_revised.py]: I'll be performing a squash merge to remove this. My bad!

File: ERA5/construct_era5_daily_cancities_revised.py (hunk 2 above)
Comment (Zeitsperre) [reply ERA5/construct_era5_daily_cancities_revised.py]: Good catch. It shouldn't be an issue for the existing data as `dly` exists, but yes, absolutely must be fixed!

**Comment by Zeitsperre** [comment]
@aulemahal How're things looking now?

**Comment by aulemahal** [comment]
They look good!

**Pull request merged**